packages/homepage/src/pages/company/index.js

```diff
@@ -232,6 +232,7 @@ export default () => (
         <TeamMember name="roman" />
         <TeamMember name="marco" />
         <TeamMember name="james" />
+        <TeamMember name="ace" />
```

Comment:
This doesn't quite work because the component will try to find the string `ace` in the names and won't find it. You can fix it in two ways:
- add your nick to the name, like `name: 'Adewale "Ace" Abati'` (which I think would be kinda cool)
- or change the `name` property to "adewale" so `<TeamMember name="adewale"`

Reply:
Thanks for the catch @gsimone.